examples/init_repo.pp

```diff
@@ -4,12 +4,6 @@
   } # RedHat
 }
 
-case $::osfamily {
-  'redhat': {
-    package { 'epel-release': }
-  } # RedHat
-}
-
```

Comment:
Intended change?

Reply:
Yes. icinga2 v2.11.0 and higher needs new boost libs and for rhel 7 and of course 6 these libs have to come from epel.

Reply:
Is described in the documentation. The docs will be split in several parts to have more better review.

Reply:
Inline docs to examples are also a good idea....

Reply:
Yep, but why is it then removed from the example? The docs tell you to add this package yourself which works that way :)

Reply:
Ops my fault. I don't know.

Reply:
I am confused. It's included in the branch.

Reply:
Damn! Yes it's correct because the current master has the package declaration twice a time.

https://github.com/Icinga/puppet-icinga2/blob/master/examples/init_repo.pp